cosmic-settings/src/pages/desktop/mod.rs

```diff
@@ -57,6 +58,7 @@ impl page::AutoBind<crate::pages::Message> for Page {
 
 #[derive(Copy, Clone, Debug)]
 pub enum Message {
+    ShowActiveWindowHint(bool),
     ShowMaximizeButton(bool),
     ShowMinimizeButton(bool),
 }
```

Comment:
May I ask if it wouldn't be better to move this to `desktop/window_management.rs` because the maximize and minimize buttons are also in the `window_management` page.

Reply:
That would need @pop-os/ux approval to deviate from the Figma design.

Reply:
Actually I'm not sure why this isn't already in the window management module